### ISSUE TITLE
Create new demo

### ISSUE THREAD

**Issue by jackfranklin** [PixelsCommander/ReactiveElements#96]
We should create a new demo that is built with Webpack, closer to how I imagine people are consuming the package.

I removed the old demo in https://github.com/PixelsCommander/ReactiveElements/commit/bdc1efce70360bc23dfb3379e8aaf1c1d1760f11

**Comment by PixelsCommander** [comment]
Any ideas of what this could be? Maybe a very thin version of a project you
use RE in? Then it might be closer to real life usage and also cheaper to
deliver.

вт, 18 дек. 2018 г. в 10:22, Jack Franklin <notifications@github.com>:

**Comment by jackfranklin** [comment]
Yeah, I've added a basic one that uses Webpack here: https://github.com/PixelsCommander/ReactiveElements/commit/f2c7bc6e3a6f287658c28964e3238ffe162ea15f.

We should build out more examples over time.

**Comment by PixelsCommander** [comment]
Yeah. Bad thing we missed to meet during RDB =( Was crazy busy, sorry.

ср, 19 дек. 2018 г. в 18:58, Jack Franklin <notifications@github.com>:

**Comment by sumalya** [comment]
Not sure where to comment hence using this thread. 
It would be very helpful if someone can explain how to include styles in the resultant web-component. After bundling using webpack, we get bundle.js and style.css. Although I am able to successfully use the bundle.js as script tag and use the <web-component /> in HTML, but since it gets injected as a shadow DOM in the DOM tree, including style.css inside <head> does not help. And I am unable to apply style.css to my web-component.

 Can someone help please?

**Comment by ShaofeiZi** [comment]
demo  error:   Cannot read property 'useShadowDom' of undefined

**Comment by wiranoid** [comment]
More demos would be great. Especially those which cover @sumalya's question about styles.
